### ISSUE TITLE
Remove unneeded files from our release zip file

### ISSUE THREAD

**Issue by acicovic** [Parsely/wp-parsely#2936]
## Is your feature request related to a problem?
Our release zip file seems to contain some files that aren't useful. For example, I think that Jest and TypeScript files (among others) are unneeded when distributing to WordPress.org.

## Describe the solution you'd like
Identify the unneeded files that get bundled in our release .zip file, and take care not to include them there.

**Comment by vaurdan** [comment]
Agreed! I think we can safely remove the `tests` directory, and all the `ts`, `tsx`, and `scss` files from the built version, since essentially those are bundled in the `build/` directory. 

Although, I'm not sure if it's against WordPress.org Plugin Directory TOS to not include the source files in the plugin, but we can investigate that. 

Thoughts?

**Comment by acicovic** [comment]
Yeah, if we risk a TOS violation, we definitely should not do it. I can try to ask around about this.
